### ISSUE TITLE
Table name contains `-` cause error.

### ISSUE THREAD

**Issue by yaquawa** [k0kubun/sqldef#17]
The error message:

```
Error 1064: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '-biz_type' at line 1
```

did you surround the table name or column name with "`" ?

**Comment by k0kubun** [comment]
Thanks to report that. I fixed that issue on v0.4.5.

**Comment by yaquawa** [comment]
@k0kubun Thanks! works like a charm now!
